lib/pacer.go

```diff
@@ -0,0 +1,170 @@
+package vegeta
+
+import (
+	"fmt"
+	"math"
+	"time"
+)
+
+// A Pacer defines the rate of hits during an Attack by
+// returning the duration an Attacker should wait until hitting
+// the next Target. If the second return value is true, the
+// attack will terminate.
+type Pacer interface {
+	Pace(elapsedTime time.Duration, elapsedHits uint64) (sleep time.Duration, stop bool)
```

Comment:
```suggestion
	Pace(elapsed time.Duration, hits uint64) (wait time.Duration, stop bool)
```

Reply:
```suggestion
	Pace(elapsed time.Duration, hitsDone uint64) (wait time.Duration, stop bool)
```
would also be a good alternative